hw/ip/aes/pre_sca/prolead/README.md

```diff
@@ -0,0 +1,265 @@
+# AES Masking Evaluation Using PROLEAD
+
+This directory contains support files to evaluate the masking employed inside the AES cipher core together with the instantiated PRNG using the tool [PROLEAD - A Probing-Based Leakage Detection Tool for Hardware and Software](https://github.com/ChairImpSec/PROLEAD).
+For further details on the tool and its capabilities, refer to the paper [PROLEAD - A Probing-Based Hardware Leakage Detection Tool](https://eprint.iacr.org/2022/965).
+
+## Prerequisites
+
+Note that this flow is experimental.
+It has been developed using Yosys 0.36 (git sha1 8f07a0d84) and sv2v v0.0.11-28-g81d8225.
+The used PROLEAD version is from Oct 31, 2023 (7ed0f9f2).
+Other versions of these tools might not be compatible.
+
+1. Download the PROLEAD tool
+   ```sh
+   git clone git@github.com:ChairImpSec/PROLEAD.git
```

Comment:
```suggestion
   git clone git@github.com:ChairImpSec/PROLEAD.git
   cd PROLEAD
   git reset --hard 7ed0f9f2
```
Maybe add this to avoid that somebody is  using the wrong version?

Reply:
Good idea :-)